### PR TITLE
Replaces the default output with the to_chat() wrapper

### PR DESCRIPTION
**Pull request by lzimann**
This will make easier for goonchat later on.

The to_chat proc is just a wrapper, needs to be changed in case of a goon chat port:
```
/proc/to_chat(target, message)
    target << message
```

Regex used:
`(\b(?!\blog|diary|bit\b)\w+\b) << ((?!browse|'|hotkey|adminhotkeys|link|other|output|sound).+?)(?:\/\/.*)?$`

The regex has issues with multilines and target.var << message, you will have to fix them manually.

### CODE REVIEW

File: code/__HELPERS/icon_smoothing.dm [hunk 1]
```diff
@@ -71,9 +71,9 @@
 		AM = find_type_in_direction(A, direction)
 		if(AM == NULLTURF_BORDER)
 			if((A.smooth & SMOOTH_BORDER))
-				adjacencies |= 1 << direction
+				adjacencies |= to_chat(1, direction)
```
Comment (Fox-McCloud) [inline code/__HELPERS/icon_smoothing.dm]: I don't think this is supposed to be a `to_chat`

File: code/game/sound.dm [hunk 2]
```diff
@@ -72,7 +72,7 @@
 		S.y = 1
 		S.falloff = (falloff ? falloff : FALLOFF_SOUNDS)
 
-	src << S
+	to_chat(src, S)
```
Comment (Cyberboss) [inline code/game/sound.dm]: No

File: code/modules/admin/admin_investigate.dm [hunk 3]
```diff
@@ -22,7 +22,7 @@
 	var/F = investigate_subject2file(subject)
 	if(!F)
 		return
-	F << "<small>[time_stamp()] \ref[src] ([x],[y],[z])</small> || [src] [message]<br>"
```
Comment (Cyberboss) [inline code/modules/admin/admin_investigate.dm]: This is a file

File: code/modules/admin/topic.dm [hunk 4]
```diff
@@ -19,17 +19,17 @@
 
 		C << 'sound/effects/adminhelp.ogg'
```
Comment (Cyberboss) [inline code/modules/admin/topic.dm]: ps_local

File: code/modules/admin/verbs/BrokenInhands.dm [hunk 5]
```diff
@@ -28,7 +28,7 @@
 	if(text)
 		var/F = file("broken_icons.txt")
 		fdel(F)
-		F << text
```
Comment (Cyberboss) [inline code/modules/admin/verbs/BrokenInhands.dm]: File

File: code/modules/admin/verbs/adminhelp.dm [hunk 6]
```diff
@@ -116,17 +116,17 @@
 		if(X.prefs.toggles & SOUND_ADMINHELP)
 			X << 'sound/effects/adminhelp.ogg'
```
Comment (Cyberboss) [inline code/modules/admin/verbs/adminhelp.dm]: again

File: code/modules/admin/verbs/adminpm.dm [hunk 7]
```diff
@@ -131,29 +131,29 @@
...
 
 			else		//recipient is an admin but sender is not
-				C << "<font color='red'>Reply PM from-<b>[key_name(src, C, 1)]</b>: [keywordparsedmsg]</font>"
-				src << "<font color='blue'>PM to-<b>Admins</b>: [msg]</font>"
+				to_chat(C, "<font color='red'>Reply PM from-<b>[key_name(src, C, 1)]</b>: [keywordparsedmsg]</font>")
+				to_chat(src, "<font color='blue'>PM to-<b>Admins</b>: [msg]</font>")
 
 			//play the recieving admin the adminhelp sound (if they have them enabled)
 			if(C.prefs.toggles & SOUND_ADMINHELP)
 				C << 'sound/effects/adminhelp.ogg'
```
Comment (Cyberboss) [inline code/modules/admin/verbs/adminpm.dm]: again

File: code/modules/admin/verbs/adminpm.dm [hunk 8]
```diff
@@ -131,29 +131,29 @@
...
-				C << "<font color='red'>Admin PM from-<b>[key_name(src, C, 0)]</b>: [msg]</font>"
-				C << "<font color='red'><i>Click on the administrator's name to reply.</i></font>"
-				src << "<font color='blue'>Admin PM to-<b>[key_name(C, src, 1)]</b>: [msg]</font>"
+				to_chat(C, "<font color='red' size='4'><b>-- Administrator private message --</b></font>")
+				to_chat(C, "<font color='red'>Admin PM from-<b>[key_name(src, C, 0)]</b>: [msg]</font>")
+				to_chat(C, "<font color='red'><i>Click on the administrator's name to reply.</i></font>")
+				to_chat(src, "<font color='blue'>Admin PM to-<b>[key_name(C, src, 1)]</b>: [msg]</font>")
 
 				//always play non-admin recipients the adminhelp sound
 				C << 'sound/effects/adminhelp.ogg'
```
Comment (Cyberboss) [inline code/modules/admin/verbs/adminpm.dm]: again

File: code/modules/admin/verbs/adminpm.dm [hunk 9]
```diff
@@ -211,9 +211,9 @@
...
 
-	C << "<font color='red' size='4'><b>-- Administrator private message --</b></font>"
-	C << "<font color='red'>Admin PM from-<b><a href='?priv_msg=[stealthkey]'>[adminname]</A></b>: [msg]</font>"
-	C << "<font color='red'><i>Click on the administrator's name to reply.</i></font>"
+	to_chat(C, "<font color='red' size='4'><b>-- Administrator private message --</b></font>")
+	to_chat(C, "<font color='red'>Admin PM from-<b><a href='?priv_msg=[stealthkey]'>[adminname]</A></b>: [msg]</font>")
+	to_chat(C, "<font color='red'><i>Click on the administrator's name to reply.</i></font>")
 	window_flash(C, ignorepref = TRUE)
 	//always play non-admin recipients the adminhelp sound
 	C << 'sound/effects/adminhelp.ogg'
```
Comment (Cyberboss) [inline code/modules/admin/verbs/adminpm.dm]: again

File: code/modules/admin/verbs/playsound.dm [hunk 10]
```diff
@@ -23,11 +23,11 @@ var/sound/admin_sound
 	admin_sound.wait = 1
 	admin_sound.repeat = 0
 	admin_sound.status = SOUND_STREAM
-		
+
 	for(var/mob/M in player_list)
 		if(M.client.prefs.toggles & SOUND_MIDI)
-			M << admin_sound
```
Comment (Cyberboss) [inline code/modules/admin/verbs/playsound.dm]: Undo the change

File: code/modules/admin/verbs/randomverbs.dm [hunk 11]
```diff
@@ -399,7 +399,7 @@ Traitors and the like can also be revived with the previous role mostly intact.
 
 	message_admins("<span class='adminnotice'>[admin] has respawned [player_key] as [new_character.real_name].</span>")
 
-	new_character << "You have been fully respawned. Enjoy the game."
```
Comment (Cyberboss) [inline code/modules/admin/verbs/randomverbs.dm]: TODO: make this use the new system

File: code/modules/events/holiday/halloween.dm [hunk 12]
```diff
@@ -69,7 +69,7 @@
 	for(var/mob/living/carbon/human/H in living_mob_list)
 		if(!H.client || !istype(H))
 			return
-		H << "<span class='danger'>Honk...</span>"
+		to_chat(H, "<span class='danger'>Honk...</span>")
 		H << 'sound/spookoween/scary_clown_appear.ogg'
 		var/turf/T = get_turf(H)
```
Comment (Cyberboss) [inline code/modules/events/holiday/halloween.dm]: ps_l

File: code/modules/events/wizard/imposter.dm [hunk 13]
```diff
@@ -54,5 +54,5 @@
 		ticker.mode.update_wiz_icons_added(I.mind)
 
 		I.log_message("<font color='red'>Is an imposter!</font>", INDIVIDUAL_ATTACK_LOG)
-		I << "<B>You are an imposter! Trick and confuse the crew to misdirect malice from your handsome original!</B>"
+		to_chat(I, "<B>You are an imposter! Trick and confuse the crew to misdirect malice from your handsome original!</B>")
 		I << sound('sound/effects/magic.ogg')
```
Comment (Cyberboss) [inline code/modules/events/wizard/imposter.dm]: ps_l

File: code/modules/mapping/swapmaps.dm [hunk 14]
```diff
@@ -383,7 +383,7 @@ swapmap
 	proc/Save()
 		if(id==src) return 0
 		var/savefile/S=mode?(new):new("map_[id].sav")
-		S << src
```
Comment (Cyberboss) [inline code/modules/mapping/swapmaps.dm]: Nope

File: code/modules/mapping/writer.dm [hunk 15]
```diff
@@ -36,7 +36,7 @@ dmm_suite{
 			fdel("[map_name].dmm")
 			}
 		var/saved_map = file("[map_name].dmm")
-		saved_map << file_text
```
Comment (Cyberboss) [inline code/modules/mapping/writer.dm]: NO

File: code/modules/mob/living/silicon/ai/say.dm [hunk 16]
```diff
@@ -160,9 +160,9 @@ var/const/VOX_DELAY = 600
 				if(M.client && !M.ear_deaf && (M.client.prefs.toggles & SOUND_ANNOUNCEMENTS))
 					var/turf/T = get_turf(M)
 					if(T.z == z_level)
-						M << voice
+						to_chat(M, voice)
 		else
-			only_listener << voice
```
Comment (Cyberboss) [inline code/modules/mob/living/silicon/ai/say.dm]: ps_l

**Review by Cyberboss — CHANGES_REQUESTED** [review]
I'm not done yet, I got up to robot defense.dm. Note some of these are just memos for myself that ill take care of in a seperate PR

File: code/modules/mapping/writer.dm (hunk 15 above)
Comment (Iamgoofball) [inline code/modules/mapping/writer.dm]: Don't worry it'll just default to X << Y

Comment (JJRcop) [inline code/modules/mapping/writer.dm]: until we change it

File: code/modules/paperwork/photography.dm [hunk 17]
```diff
@@ -364,7 +364,7 @@
 	P.fields["blueprints"] = blueprintsinject
 
 	aipictures += P
-	usr << "<span class='unconscious'>Image recorded</span>"	//feedback to the AI player that the picture was taken
```
Comment (lzimann) [inline code/modules/paperwork/photography.dm]: comment deleted

File: code/modules/power/apc.dm [hunk 18]
```diff
@@ -446,65 +446,65 @@
 	else if	(istype(W, /obj/item/weapon/screwdriver))	// haxing
 		if(opened)
 			if (cell)
-				user << "<span class='warning'>Close the APC first!</span>" //Less hints more mystery!
```
Comment (lzimann) [inline code/modules/power/apc.dm]: comment deleted

File: code/modules/paperwork/photography.dm (hunk 17 above)
Comment (Iamgoofball) [inline code/modules/paperwork/photography.dm]: I have regex to preserve comments somewhere

File: code/modules/station_goals/dna_vault.dm [hunk 19]
```diff
@@ -87,35 +87,35 @@ var/list/non_simple_animals = typecacheof(list(/mob/living/carbon/monkey,/mob/li
 		if(!H.myseed)
 			return
 		if(!H.harvest)// So it's bit harder.
-			user << "<span clas='warning'>Plant needs to be ready to harvest to perform full data scan.</span>" //Because space dna is actually magic
```
Comment (lzimann) [inline code/modules/station_goals/dna_vault.dm]: comment deleted

File: code/world.dm [hunk 20]
```diff
@@ -267,7 +267,7 @@
 /world/proc/save_mode(the_mode)
 	var/F = file("data/mode.txt")
 	fdel(F)
-	F << the_mode
```
Comment (lzimann) [inline code/world.dm]: file

File: code/modules/paperwork/photography.dm (hunk 17 above)
Comment (lzimann) [inline code/modules/paperwork/photography.dm]: my regex just ignored it, but ended up removing entirely(also fucking up links)

File: code/modules/mob/living/carbon/human/species.dm [hunk 21]
```diff
@@ -664,15 +664,15 @@
...
+					to_chat(H, "<span class='warning'>You need a suit before you can attach this [I.name]!</span>")
 				return 0
 			if(!H.wear_suit.allowed)
 				if(!disable_warning)
-					H << "You somehow have a suit with no defined allowed items for suit storage, stop that."
+					to_chat(H, "You somehow have a suit with no defined allowed items for suit storage, stop that.")
 				return 0
 			if(I.w_class > WEIGHT_CLASS_BULKY)
 				if(!disable_warning)
-					H << "The [I.name] is too big to attach."  //should be src?
```
Comment (lzimann) [inline code/modules/mob/living/carbon/human/species.dm]: comment deleted

File: code/modules/mob/living/silicon/ai/say.dm [hunk 22]
```diff
@@ -160,9 +160,9 @@ var/const/VOX_DELAY = 600
 				if(M.client && !M.ear_deaf && (M.client.prefs.toggles & SOUND_ANNOUNCEMENTS))
 					var/turf/T = get_turf(M)
 					if(T.z == z_level)
-						M << voice
```
Comment (lzimann) [inline code/modules/mob/living/silicon/ai/say.dm]: this and the one below are sounds gotta change them back

**Comment by lzimann** [comment]
Alright this should be good to go now.

**Review by Cyberboss — APPROVED** [review]

**Comment by Robustin** [comment]
I updated my branch to master and I'm getting hundreds of errors from this what the fuck how I get my branch is usable condition again ahHHHH

**Comment by Xhuis** [comment]
@Robustin 

If you're using DM:
1. Go to Find and Replace (Ctrl+H)
2. Copypaste the regex in the PR into the Find box
3. Put `to_chat($1, $2)` in the Replace box
4. Set to Open Files and open all your changed files -OR- set to Included Files
5. Let 'er rip! You will have to fix some issues yourself.
